django/applications/catmaid/control/useranalytics.py

```diff
@@ -426,7 +426,7 @@ def generateReport(user_id, project_id, activeTimeThresh, start_date, end_date,
 
     # Bottom left plot: net active time per day
     ax2 = plt.subplot2grid((2,2), (1,0))
-    ax2.bar( at_timeaxis, netActiveTime, color='k', align='edge')
+    ax2.bar(at_timeaxis, netActiveTime, color='k', align='edge')
```

Comment:
not sure where all these weird spaces come from

Reply:
Yeah, there seemed to have been quite a few of them.